### DISCUSSION TITLE
wildcards have disappared after a update?

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2235] [Q&A] [answered]
just loading and said something about using a old model of juggernautXL_v8Rundiffusion.safetensors
asked me to add the some like --aways update model ( code like that )

re downloaded some update
now i have re loaded and the wildcard tab has gone?

before
![Screenshot 2024-02-10 at 14-51-25 Fooocus 2 1 862](https://github.com/lllyasviel/Fooocus/assets/35565479/ed2610ce-3487-433e-a64d-e139e47264dd)

now
![screenshot-127 0 0 1_7865-2024 02 11-11_40_35](https://github.com/lllyasviel/Fooocus/assets/35565479/04b8a6e8-23e8-4be9-b704-0a00e236ae0a)

**Comment by mashb1t** [comment, marked as answer]
@lgwjames you were using changes of https://github.com/lllyasviel/Fooocus/pull/1640, which you reset to do the update (i assume).
Feel free to re-apply the changes of mentioned PR as it has not yet been merged into main.

**Reply by lgwjames** [reply]
ill have to remember how i did that lol
